src/library/vm/vm.cpp

```diff
@@ -317,6 +318,36 @@ void vm_obj_cell::dealloc() {
     }
 }
 
+unsigned hash(vm_obj const & o) {
+    unsigned int h = 555; // just a seed for the hash
```

Comment:
Yeah I'm not sure what to do about that. The use case I have for it is always a small element. Maybe instead of implementing a hashing for arbitrary vm objects I should just do a heuristic hash within the widget code that finishes after a depth of 3.

Reply:
No, it's fine.  I just wanted to make you aware of the issue.  We already have the problem elsewhere: #151